Source/Core/Core/HW/EXI/EXI_Device.h

```diff
@@ -89,14 +89,14 @@ struct fmt::formatter<ExpansionInterface::EXIDeviceType>
       // i18n: A mysterious debugging/diagnostics peripheral for the GameCube.
       _trans("AD16"),
       _trans("Microphone"),
-      _trans("Broadband Adapter (TAP)"),
```

Comment:
Uh, hold on. You can't just remove this one.

Reply:
Should be resolved now.  Was a silly misunderstanding with how this worked.